# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../binutils/have_stdlib.patch
# Copyright (C) 2004 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- binutils-2.18.50.0.8/libiberty/cplus-dem.c.vanilla	2008-07-16 20:29:10.000000000 +0200
+++ binutils-2.18.50.0.8/libiberty/cplus-dem.c	2008-07-16 20:29:27.000000000 +0200
@@ -49,12 +49,7 @@
 #include <string.h>
 #include <stdio.h>
 
-#ifdef HAVE_STDLIB_H
 #include <stdlib.h>
-#else
-void * malloc ();
-void * realloc ();
-#endif
 
 #include <demangle.h>
 #undef CURRENT_DEMANGLING_STYLE
